pkg/steps/optional_operators.go

```diff
@@ -63,6 +67,9 @@ func resolveOptionalOperator(parameters getter) (*optionalOperator, error) {
 		*valuePointer = value
 	}
 
+	if oo.Bundle, err = parameters.Get(OOBundle); err != nil {
+		oo.Bundle = ""
+	}
```

Comment:
Hm, from the changes I saw in the registry I'd say the bundle would be required and not optional, no? It's consumed by the new scorecard step which would be always present, so withtout a bundle the test would always fail....

Reply:
That's a good point - adding the change to make it required.

Reply:
@petr-muller I made the changes to have the bundle required, ptal